loop-data/tool.gpt

```diff
@@ -0,0 +1,8 @@
+Name: Loop Data
+Description: Provide either the data_list or dataset_id parameter, but not both.
+Metadata: index: false
```

Comment:
I think if we move this tool to the `system` section of `index.yaml`, then we can get rid of this.